cloud/wfm-rest-api/src/impl/ApiController.ts

```diff
@@ -95,6 +95,12 @@ export class ApiController<T> {
       return Bluebird.reject(error);
     }
 
+    req.body.id = req.body.id || req.params.id;
```

Comment:
Let's have `PUT /:id`, it's the regular way to go since it's a request to modify the resource at the endpoint to a specific entity.

see section 9.6 [here](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html)